### PR TITLE
Moving executor to service

### PR DESCRIPTION
**Pull request by Numbers88s**
## 🤔  Description

Moving executor functionality to domain driven service architecture

## 🚀  GIF
_What gif best describes this PR or how it makes you feel?_

hide

### CODE REVIEW

**Review by efritz — APPROVED** [review]
:yes:

File: internal/database/database.go [hunk 1]
```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"database/sql"
 
+	executor "github.com/sourcegraph/sourcegraph/cmd/frontend/services/executors/store"
+	executorDB "github.com/sourcegraph/sourcegraph/cmd/frontend/services/executors/store/db"
```
Comment (eseliger) [inline internal/database/database.go:8]: We can't import from a `cmd` package in `internal`. This needs to move into an internal package as well. While we're at this: It'd be nice if executors code could become enterprise code but not a blocker :)

**Review by eseliger — CHANGES_REQUESTED** [review]

File: internal/database/database.go (hunk 1 above)
Comment (efritz) [reply internal/database/database.go:8]: Thanks for catching this. @Numbers88s if you have trouble breaking this import down I can help or we could pair on this to get it over the 🏁 .

Comment (Numbers88s) [reply internal/database/database.go]: @eseliger I tried moving it to enterprise but it was too much for right now, so I'll circle back to it. I did removed the import from `cmd` however now the codeinsights db is not passing its test 🙃 even thought I don't touch any of the insights code. What's the relationship here?

Comment (eseliger) [reply internal/database/database.go]: I think that's a CI flake, we've seen these a couple times before.. Retrying might help 🙈

Comment (Numbers88s) [reply internal/database/database.go]: Oh cool @eseliger, yeah that was it. It passed CI 🎉 .

**Comment by Numbers88s** [comment]
Sorry y'all some git foo went awry. Closing this one and opening a clean one.

**Pull request closed without merging**